addons/common/CfgVehicles.hpp

```diff
@@ -5,4 +5,97 @@ class CfgVehicles {
         scope = 1;
         displayName = "";
     };
+
+    class Module_F;
+
+    class ModuleDate_F: Module_F {
+        class Arguments {
+            class Year {
+                class values {
+                    YEAR(1900);
+                    YEAR(1901);
+                    YEAR(1902);
+                    YEAR(1903);
+                    YEAR(1904);
+                    YEAR(1905);
+                    YEAR(1906);
+                    YEAR(1907);
+                    YEAR(1908);
+                    YEAR(1909);
+                    YEAR(1910);
+                    YEAR(1911);
+                    YEAR(1912);
+                    YEAR(1913);
+                    YEAR(1914);
+                    YEAR(1915);
+                    YEAR(1916);
+                    YEAR(1917);
+                    YEAR(1918);
+                    YEAR(1919);
+                    YEAR(1920);
+                    YEAR(1921);
+                    YEAR(1922);
+                    YEAR(1923);
+                    YEAR(1924);
+                    YEAR(1925);
+                    YEAR(1926);
+                    YEAR(1927);
+                    YEAR(1928);
+                    YEAR(1929);
+                    YEAR(1930);
+                    YEAR(1931);
+                    YEAR(1932);
+                    YEAR(1933);
+                    YEAR(1934);
+                    YEAR(1935);
+                    YEAR(1936);
+                    YEAR(1937);
+                    YEAR(1938);
+                    YEAR(1939);
+                    YEAR(1940);
+                    YEAR(1941);
+                    YEAR(1942);
+                    YEAR(1943);
+                    YEAR(1944);
+                    YEAR(1945);
+                    YEAR(1946);
+                    YEAR(1947);
+                    YEAR(1948);
+                    YEAR(1949);
+                    YEAR(1950);
+                    YEAR(1951);
+                    YEAR(1952);
+                    YEAR(1953);
+                    YEAR(1954);
+                    YEAR(1955);
+                    YEAR(1956);
+                    YEAR(1957);
+                    YEAR(1958);
+                    YEAR(1959);
+                    YEAR(1960);
+                    YEAR(1961);
+                    YEAR(1962);
+                    YEAR(1963);
+                    YEAR(1964);
+                    YEAR(1965);
+                    YEAR(1966);
+                    YEAR(1967);
+                    YEAR(1968);
+                    YEAR(1969);
+                    YEAR(1970);
+                    YEAR(1971);
+                    YEAR(1972);
+                    YEAR(1973);
+                    YEAR(1974);
+                    YEAR(1975);
+                    YEAR(1976);
+                    YEAR(1977);
+                    YEAR(1978);
+                    YEAR(1979);
+                    YEAR(1980);
+                    YEAR(1981);
```

Comment:
```suggestion
                    YEAR(1981);
                    class Number1982 {};
```
This should fix ordering.

Reply:
It does not appear to do so